api/src/routes/post/Post.ts

```diff
@@ -188,9 +198,10 @@ router.get(
  *         description: The updated content of the post.
  */
 router.patch(
-    '/:postId',
+    '/:slug',
     ...updatePostRequestValidators,
```

Comment:
Change this to editPostRequestValidators so we are keeping our naming consistent